upload.go

```diff
@@ -302,7 +302,13 @@ func (ua *uploadAction) uploadBuild(retryAllowed bool) (bool, error) {
 
 	defer file.Close()
 
-	client := &http.Client{}
+	client := &http.Client{
+		Transport: &http.Transport{
+              		TLSHandshakeTimeout: 0,
+              		ResponseHeaderTimeout: 0,
+              		ExpectContinueTimeout: 0,
+            	},
+	}
```

Comment:
nit: I’m not a big fan of extraneous commas in arrays and structs (like you have on lines 309 and 310).